Katello/katello
# Fixes #29751 - Pulp 3 incremental update

Pulp 3 incremental update support for Yum repositories.  To test, try doing various incremental update commands just like you would with Pulp 2.

Note, since this is with Pulp 3 don't forget to enable `:katello_applicability:` in the config.

Potential test steps:

1) Create a content view with yum content
2) Filter out some errata and RPMs
3) Run various incremental update commands.  Incremental update lets you make a new content view version with extra RPMs and errata that you specify.  For example:
```
hammer content-view version incremental-update --organization "Default Organization" --content-view-version-id 148 --errata-ids 5,13 --package-ids 64,69
```
You should notice that dependency solving is on, like with Pulp 2.

You can also try applying errata from the Content -> Errata page.

Questions:
1) Is it okay that the added content is being calculated from the content view version rather from the Pulp task output?  This seemed necessary since the copy task doesn't always result in a new Pulp repository version, which would cause no content to be reported.
